### PR TITLE
[BX-1218][BX-1238] NFT Context Menus

### PR DESCRIPTION
**Pull request by derHowie**
## What changed (plus any additional context for devs)
Added context menus to nft gallery items

## Screen recordings / screenshots
PoW: https://recordit.co/tBfImOZJu7

## What to test
Ensure that the new context menus work. Make sure they don't interfere with keyboard navigation and that the context shortcut (SPACE) works when navigation to each gallery item.

### CODE REVIEW

**Review by estebanmino — COMMENTED** [review]
when clicking on `Explorer` for base nfts i'm getting 404s

**Comment by derHowie** [comment]
> when clicking on `Explorer` for base nfts i'm getting 404s

This appears to have been broken from the start. It's now fixed.

File: src/entries/popup/pages/home/NFTs/NFTContextMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+}: {
+  children: ReactNode;
+  nft?: UniqueAsset | null;
+}) {
+  const hasContractAddress = !!nft?.asset_contract.address;
+  const hasNetwork = !!nft?.network;
+
+  const explorerTitle =
+    nft?.network === 'mainnet' ? 'Etherscan' : i18n.t('nfts.details.explorer');
+  const getBlockExplorerUrl = () => {
```
Comment (brunobar79) [inline src/entries/popup/pages/home/NFTs/NFTContextMenu.tsx:34]: I'd memoize this fn

File: src/entries/popup/pages/home/NFTs/NFTDropdownMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+  children: ReactNode;
+  nft?: UniqueAsset | null;
+}) {
+  const hasContractAddress = !!nft?.asset_contract.address;
+  const hasNetwork = !!nft?.network;
+
+  const explorerTitle =
+    nft?.network === 'mainnet' ? 'Etherscan' : i18n.t('nfts.details.explorer');
+
+  const getBlockExplorerUrl = () => {
```
Comment (brunobar79) [inline src/entries/popup/pages/home/NFTs/NFTDropdownMenu.tsx:37]: Same code as earlier. Maybe utils?

**Review by brunobar79 — APPROVED** [review]

**Review by estebanmino — APPROVED** [review]
yep working now

**Pull request merged**